### PR TITLE
Change name to IBM FHIR Server #114

### PR DESCRIPTION
**Pull request by prb112**
Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

**Comment by prb112** [comment]
I changed to &reg; for html encoding in the swagger/openapi generator.

File: fhir-install/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# FHIR Server Install
+# IBM FHIR Server Install
 
-Running FHIR Server as a Docker container.
+Running IBM FHIR Server as a Docker container.
```
Comment (lmsurpre) [inline fhir-install/README.md]: extremely minor, not really related to your edit
```suggestion
Running the IBM FHIR Server as a Docker container.
```

File: fhir-install/src/main/resources/docs/README.txt [hunk 2]
```diff
@@ -1 +1,5 @@
-The FHIR Server User Guide has moved to https://github.com/ibm/fhir/docs
\ No newline at end of file
+The IBM FHIR Server User Guide has moved to https://ibm.github.com/FHIR/
+
+The source is available at https://github.com/ibm/fhir/docs/
```
Comment (lmsurpre) [inline fhir-install/src/main/resources/docs/README.txt]:
```suggestion
The source is available at https://github.com/IBM/FHIR/tree/master/docs
```

**Comment by prb112** [comment]
both are easy fies

**Pull request merged**